app/src/pages/inside/productVersionPage/linkedTestCasesTab/tagList/tagList.jsx

```diff
@@ -46,21 +47,56 @@ export const TagList = ({ tags }) => {
     setCount(overflowedElements);
   };
 
+  const getFullWidthOffset = () => {
+    const parentElement = listRef.current;
+    const hiddenSet = new Set();
+
+    if (!parentElement) return;
+
+    const tagElements = [...parentElement.children].filter(
```

Comment:
Provide a clear variable name that describes the output of the filtering logic. As I understand this is some tag elements without buttons

Reply:
renamed